.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish GPR Package
```

Comment:
I think we should just remove this.  This was me testing what this would look like as a published package.  I am not sure there is value in this.

Reply:
Makes sense. I was curious what the package was for.

Reply:
Yeah, it was a bit weird, but I was publishing the entire dataset as a package where the wrapper around `@pkgjs/statusboard` was the interface.  I guess my thought was that something in theory could consume the statusboard data from multiple communities and pull them together.  For example if the OpenJS Foundation wanted to have a board with all the project activity together, they could import all of the individual projects and build a view from that.  I still think it is an interesting idea, but of little value to us today.

Reply:
Ah. That is an interesting idea, though, especially with the amount of time it takes to generate the data just for our repos, it would make sense to have a static copy to build off of. But yea, agree no use to us currently. But fun to experiment with the package publishing I'm sure 😄

Reply:
> it would make sense to have a static copy to build off of

Yeah that is what the published package is.  It is literally the leveldb database 🤪